commit 74d048e77e1699c6e7282472bc6bb59e2e32c05e (from 2d2c9bdf92d840fd5ed474d0107257f18453c50b)
Merge: 2d2c9bd 4afa293
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Sat Oct 1 04:52:56 2016 -0500

    Merge pull request #4927 from thinkyhead/rc_bltouch_noraise
    
    Don't check Z_MAX on raise when the probe pwns the pin

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 39bbb59..f334341 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -2945,10 +2945,10 @@ inline void gcode_G4() {
     SERIAL_ECHOPGM("Probe: ");
     #if ENABLED(FIX_MOUNTED_PROBE)
       SERIAL_ECHOLNPGM("FIX_MOUNTED_PROBE");
-    #elif HAS_Z_SERVO_ENDSTOP
-      SERIAL_ECHOLNPGM("SERVO PROBE");
     #elif ENABLED(BLTOUCH)
       SERIAL_ECHOLNPGM("BLTOUCH");
+    #elif HAS_Z_SERVO_ENDSTOP
+      SERIAL_ECHOLNPGM("SERVO PROBE");
     #elif ENABLED(Z_PROBE_SLED)
       SERIAL_ECHOLNPGM("Z_PROBE_SLED");
     #elif ENABLED(Z_PROBE_ALLEN_KEY)

